### PR TITLE
reduce bloat from assert code

### PR DESCRIPTION
**Pull request by WalterBright**
By switching from a dynamic array to a static array, this reduces bloat generated for assert() functions.

### CODE REVIEW

**Comment by yebblies** [comment]
Why is win64 different?

**Comment by WalterBright** [comment]
Win64 has a wacky calling convention that only allows 64 bit parameters. Applying the same fix to Win64 involves creating stack temporaries and other stuff, which might be worthwhile doing later.

**Comment by andralex** [comment]
noice

File: src/glue.c [hunk 1]
```diff
@@ -1533,23 +1533,43 @@ Symbol *toSymbol(Type *t)
...
-        dtsize_t(&dt, len);
-        dtabytes(&dt,TYnptr, 0, len + 1, id);
 
-        m->sfilename = symbol_generate(SCstatic,type_fake(TYdarray));
+        if (config.exe == EX_WIN64)
+        {
+            // Put out as a dynamic array
+            dtsize_t(&dt, len);
+            dtabytes(&dt,TYnptr, 0, len + 1, id);
+
```
Comment (andralex) [inline src/glue.c]: uhm these arbitrary newlines grind my gears

Comment (WalterBright) [inline src/glue.c]: I like to logically group things that way. Sorry they grind your gears!

**Pull request merged**